### ISSUE TITLE
SAFplus installer for doesn't compile tipc-config with the right path

### ISSUE THREAD

**Issue by karthick18** [OpenClovis/SAFplus-Availability-Scalability-Platform#39]
SAIC had this issue where the install of SAFplus failed during the build of tipc-config.
It was found that tipc-config is being built with: export KERNELDIR to reflect the path while building tipc-config for tipcutils-1.1.9.
This is incorrect as the right include path or command for building TIPC config is:
make INCLUDE_PATH=/lib/modules/`uname -r`/build/include VERSION="shipped-tipc-version"
Replace uname -r with the kernel version on the target.
Replace "shipped-tipc-version" with 1.7.7 or whatever version is in the installer bundle. (from net/tipc/tipc_core.h --> TIPC_MOD_VER reflects the same)

The workaround is to copy the tipc include files from the kernel source to the standard system wide include path in /usr/include/linux
So cp -v /lib/modules/`uname -r`/build/include/linux/tipc*.h /usr/include/linux
is a simple workaround that unblocked the customer. But we can still try and avoid it by fixing the installer to use the right command for building tipc-config.
Assigning it to @hoangle who can it assign it to the right guy as this needs some testing to confirm/close and I can't do that here.
Keeping @AndrewStoneOpenClovis  in the loop as well.

Customer is unblocked with the workaround. So this is low priority but needs to be fixed before the next SAFplus release/update which might re-introduce the install failure for the customer.
